unittests/ASM/VEX/vtestps.asm

```diff
@@ -12,7 +13,7 @@
 
 ; Uses AX and BX and stores result in r15
```

Comment:
Perhaps remove the r15 bit of the comment, since this is no longer true

Reply:
You're right, I'll change that in a bit.